integrations/pjrt/CMakeLists.txt

```
@@ -52,6 +52,10 @@ add_subdirectory("${IREE_ROOT_DIR}" "iree_core" EXCLUDE_FROM_ALL)
 # toolchain level.
 iree_setup_toolchain()
 
+# Setup protoc and protobuf library
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

We use `CMAKE_CURRENT_LIST_DIR` in other places: https://github.com/iree-org/iree/blob/d48071df356eec533554699ac1e658f6809aacbe/CMakeLists.txt#L542-L544

May want to do the same here:

```suggestion
list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
```

However, the example at https://cmake.org/cmake/help/latest/variable/CMAKE_MODULE_PATH.html uses `CMAKE_CURRENT_SOURCE_DIR`.

More discussion: https://stackoverflow.com/questions/15662497/difference-between-cmake-current-source-dir-and-cmake-current-list-dir

(I don't think it functionally matters here, but consistency is nice)

---

Good to know this variable. I'll update this part.